docs/team/lowkaiwei98.md

```diff
@@ -5,27 +5,27 @@ title: Kaiwei's Project Portfolio Page
 
 ### Project: Trackermon
 
-Trackermon - The app will help to allow the user to track and remember what shows they have watched. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java.
-
+Trackermon is a desktop application for tracking and managing shows, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). Trackermon allows people who want to track and remember what shows they have watched, are currently watching, or plan to watch.
 Given below are my contributions to the project.
 
-* **New Feature**:
-  to be added soon
+**New Feature**: Added the ability to sort the show list.
+* What it does: allows the user to sort the list by name, status, tag, rating and the ability to change the priority of the sort. 
+* Justification: This feature improves the product significantly because a user can sort the list and view it in an organised matter. Without this feature, the list might remain a mess, making it less appealing to the user. 
 
-* **Code contributed**:
-  to be added soon
+**Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=lowkaiwei98&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-02-18)
 
-* **Project management**:
-  to be added soon
+**Project management**:
+  * Managed issues relating to Sort, Edit, Clear, etc. [\#118] (https://github.com/AY2122S2-CS2103T-T09-3/tp/issues/118) [\#103] (https://github.com/AY2122S2-CS2103T-T09-3/tp/issues/103)
+  * Bug test and asked teammates to fix for bugs relating to their code [\#145] (https://github.com/AY2122S2-CS2103T-T09-3/tp/issues/145)
```

Comment:
This should be `Tested for bugs and asked teammates to fix bugs relating to their code`

Reply:
ok